frontend/public/components/dashboard/dashboards-page/overview-dashboard/inventory-card.tsx

```diff
@@ -121,14 +123,25 @@ export const InventoryCard = connectToFlags(
         <DashboardCardTitle>Cluster Inventory</DashboardCardTitle>
       </DashboardCardHeader>
       <DashboardCardBody>
-        <ClusterInventoryItem model={NodeModel} mapper={getNodeStatusGroups} />
-        <ClusterInventoryItem model={PodModel} mapper={getPodStatusGroups} />
-        <ClusterInventoryItem model={StorageClassModel} />
         <ClusterInventoryItem
           model={PersistentVolumeClaimModel}
           mapper={getPVCStatusGroups}
           useAbbr
         />
```

Comment:
To preserve the order of the ClusterInventoryItems, this should go after StorageClassModel ClusterInventoryItem. Also `data-test-id` prop should get added to this  item as well.

Reply:
Done